src/timedata.cpp

```diff
@@ -79,7 +74,8 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
         int64_t nMedian = vTimeOffsets.median();
         std::vector<int64_t> vSorted = vTimeOffsets.sorted();
         // Only let other nodes change our time by so much
-        if (abs64(nMedian) <= std::max<int64_t>(0, gArgs.GetArg("-maxtimeadjustment", DEFAULT_MAX_TIME_ADJUSTMENT))) {
+        int64_t max_adjustment = std::max<int64_t>(0, gArgs.GetArg("-maxtimeadjustment", DEFAULT_MAX_TIME_ADJUSTMENT));
+        if (nMedian < -max_adjustment || nMedian > max_adjustment) {
```

Comment:
I think this is flipped. Now if `nMedian` is beyond the max `nTimeOffset` is set to `nMedian`, whereas it was the opposite before:

Example: max = 4, nMedian = -3
Before: abs64(-3) <= 4 == 3 <= 4 == true
Now: -3 < -4 || -3 > 4 == false

Reply:
Uh, fixed. Thanks!